scripts/hca/README.md

```diff
@@ -0,0 +1,33 @@
+## Building the code
+
+1. Create a Python Virtual Environment for so some lines of code can run.
+```sh
+python3 -m venv .env
+```
+2. ```.env/bin/activate``` engages the environment so  pip and black tqm can be installed.
+```sh
+source .env/bin/activate
+```
+3. install python dependencies
+```sh
+pip install --upgrade pip
+pip install black tqdm
+```
+4. generate and save a AUTH_TOKEN which is required to make API requests.
+```sh
+AUTH_TOKEN="Authorization: Bearer $(gcloud auth application-default print-access-token)"
+```
+5. run python code to extract the project and snapshot data as json files. ```create-hca-collection``` combines the two to create a collection file
+```sh
+python3 extract-hca-projects.py
+python3 extract-hca-snapshots.py
+python3 create-hca-collection
+```
+
+6. ```.env/bin/activate``` must be used to create an environment so  ```ingest-hca-collection``` script can run and to avoid a tpq error.
+```sh
+source .env/bin/activate
+python3 ingest-hca-collection.py
+```
```

Comment:
```suggestion
6. `.env/bin/activate` must be used to activate the virtual env to avoid a `tqdm` error in `ingest-hca-collection.py`.

```sh
source .env/bin/activate
python3 ingest-hca-collection.py
```

```